Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md

```diff
@@ -737,7 +742,7 @@ Hover over a point in the chart to view the quantile percentage and Kolmogorov-S
 
 For the K-S chart above, if we look at the top 60% of the data, the at-chance model (the dotted diagonal line) tells us that only 60% of the data was successfully separate between positives and negatives (survived and did not survived). However, with the model, it was able to do .499, or about 50% of the cases were successfully separated between positives and negatives.
```

Comment:
where did you get .499?

Reply:
Updated the value to .4091 or about 41%. Thanks for catching it.